app/controllers/catalog_controller_decorator.rb

```diff
@@ -62,7 +62,13 @@
     { prop => CatalogController.send(:index_options, prop, DogBiscuits.config.property_mappings[prop]) }
   end
   CatalogController.send(:add_index_field, config, index_props)
-  config.add_index_field 'all_text_tsimv', label: "Item contents", highlight: true, helper_method: :render_ocr_snippets, if: :query_present?
+  # this is identical to what is in Hyku but the dog biscuits code seems to mean it doesn't carry through
```

Comment:
instead of copying this over you can subtract it some place I saw. Let me find it ...

Reply:
I think you can add it [here](https://github.com/scientist-softserv/adventist_knapsack/blob/3560ec048061c0610afbec232fde5ec2cede970a/app/controllers/catalog_controller_decorator.rb#L48). 

`next if key == 'all_text_tsimv' and it'll fall back to hyku`

Reply:
Good suggestion, thanks! I'll make that change.

Reply:
Actually it is already there (line 49), and it still wasn't working, so I think this is needed.